rotkehlchen/types.py

```diff
@@ -523,3 +510,16 @@ def deserialize(cls: Type['AddressbookEntry'], data: Dict[str, Any]) -> 'Address
 class AddressbookType(SerializableEnumMixin):
     GLOBAL = 1
     PRIVATE = 2
+
+
+class DataImportSource(SerializableEnumMixin):
```

Comment:
you do not need to change the names just because you introduced an enum. In fact kelsos will get pissed if you got from snake case to space separated words as it breaks consistency.

I would suggest having a different serialize() function here for the api. And maybe we can even slowly move such a function that keeps the snake case down to `SerializableEnumMixin`.

So one serialization being for human consumption (with spaces) and one for machine (with snake case).

Or is that just silly @kelsos and you can adjust to whatever?

Reply:
Yes I would prefer whatever is used in the API to be **snake_case** especially if that something is used as a key anywhere in JSON

```json
{
  "space separated": 1
}
```
Looks silly.

Reply:
Done